src/AppBundle/Entity/FacebookVideo.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="facebook_videos")
+ */
+class FacebookVideo
+{
+    use EntityTimestampableTrait;
+
+    /**
+     * @ORM\Column(type="bigint")
```

Comment:
Bigint? Any valid reasons why allocating such a type for storing a couple of video records?